formbricks/formbricks
# fix: overflowing CodeBlock in "Share or embed your survey" modal

## What does this PR do?

This PR fixes issue related to  overflowing CodeBlock in "Share or embed your survey" modal mentioned in https://github.com/formbricks/formbricks/issues/1403

Fixes [# (issue)](https://github.com/formbricks/formbricks/issues/1403)

![image](https://github.com/formbricks/formbricks/assets/45624147/0abc8bb4-61a8-4aba-a9b7-17011362354d)
![image](https://github.com/formbricks/formbricks/assets/45624147/3da97235-9372-49ea-aec2-d27428513029)

## Type of change




## How should this be tested?



1. Make an in-app survey with multiple questions
    try to upload images as image URL's are huge to fit in CodeBlock's normal width
2. Save Form and click on Share or Publish
3. Will open "Share or embed your survey" modal
4. Click on Embed in an Email tab
5. then click on View Embed Code tab
6. try to pay around with displayed codeblock and try to scroll

Entire Code should be visible and readable from CodeBlock with simple scroll operation if needed

## Checklist



### Required


### Appreciated
